pkg/edit/complete/generators.go

```diff
@@ -108,14 +107,15 @@ func generateFileNames(seed string, onlyExecutable bool) ([]RawItem, error) {
 	// Make candidates out of elements that match the file component.
 	for _, info := range infos {
 		name := info.Name()
+		finfo, _ := info.Info()
```

Comment:
This can return an error; do not ignore it.

Having two variables named "info" and "finfo" is confusing. Better rename the original info variable as it is no longer a FileInfo value.

Reply:
Yes, it can return an error. But ignoring it in this context should be okay; although, the logic is subtle. Since the `FileInfo` object will be initialized to the zero value(s) the property tests will all evaluate to false causing it to be silently ignored. In hindsight I agree that an explicit `if err != nil { continue }` is clearer and thus preferable.